ui/js/controllers/perf/compare.js

```diff
@@ -1,10 +1,34 @@
 "use strict";
 
+perf.value('compareDefaultTimeRange', 86400 * 2);
+
+perf.factory('compareFactory',[
+    '$rootScope', 'ThResultSetModel',
+    function ($rootScope, ThResultSetModel) {
+        return {
+            verifyRevision : function(project, revision, rsid){
```

Comment:
Pay attention to the spacing: there should be a space between ")" and "{" (also between verifyRevision and ':', though you'll be taking that out)